support-frontend/assets/components/recaptcha/recaptcha.jsx

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from 'react';
+import './recaptcha.scss';
+
+export function Recaptcha() {
+  return (
+    <>
+      <div id="robot_checkbox" className="robot_checkbox" />
+      <p className="recaptcha-terms ">
+        By ticking this box, you agree to let Google check if you are human. Please refer to their <a href='https://policies.google.com/terms'>Terms</a> and <a href='https://policies.google.com/privacy'>Privacy</a> policies.
```

Comment:
The copy should be: "By ticking this box, you agree to let Google perform a security check to confirm you are a human. Please refer to their Terms and Privacy policies"

Reply:
Thanks for flagging this @paulbrown1982, I've corrected the copy now 🙂